extensions/interactions/DragAndDropSortInput/directives/oppia-interactive-drag-and-drop-sort-input.component.ts

```diff
@@ -242,11 +250,17 @@ export class InteractiveDragAndDropSortInputComponent implements OnInit {
   }
 
   getContentIdOfHtml(html: string): string {
+    let contentId = this.choicesValue[this.choices.indexOf(html)].contentId;
+
+    if (contentId === null) {
+      throw new Error('contentId cannot be null');
+    }
     // Returns the content id of the html.
-    return this.choicesValue[this.choices.indexOf(html)].contentId;
+    return contentId;
   }
 
-  getHtmlOfContentId(contentId: string): string {
+  // Returns undefined if the content id is not found.
```

Comment:
Please elaborate this comment a bit more

Reply:
Done